charts/images.yaml

```diff
@@ -1,8 +1,8 @@
 images:
 - name: calico-node
   sourceRepository: github.com/projectcalico/calico
-  repository: quay.io/calico/node
-  tag: v3.16.4
+  repository: eu.gcr.io/gardener-project/3rd/calico/node
```

Comment:
Where are these images coming from / how are they published here?

Reply:
I uploaded them manually, they switched from `quay.io` to `docker`.

Reply:
I believe this should be not a problem and we should continue using the upstream docker images.

Reply:
We run into rate limits if we use the upstream images.

Reply:
I don't get, they should run with "pullPolicy: IfNotPresent" and this should not be a problem.

Reply:
I run even locally into rate limits, I don't care using them but I would personally feel better knowing that we don't run into that issue.

Reply:
We did similar workaround for golang and alpine images as they are used to build the extension images because of an issue with the ci/cd that is always pulling images and is not able to cache the images AFAIK.

Reply:
Then I will adapt it.

Reply:
Thank you!

Reply:
Upgraded the PR :)